src/pages/VideoListing.js

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios'
+import React, { useEffect, useState } from 'react'
+import { VideoCard } from '../components'
+import { SearchCard } from '../components/Search/Search'
+import { errorToast } from '../components/Toast/Toast'
+import { DATA_FROM_SERVER } from '../constants/reducerConstants'
+import { useVideo } from '../context/VideoContext'
+export const VideoListing = () => {
+  const [isLoading, setIsLoading] = useState(false)
+  const { videoState, videoDispatch } = useVideo()
+  const searchResult = (videos, search) => {
+    if (search) {
+      videos = [...videos].filter((video) =>
+        video.title.toLowerCase().includes(search)
+      )
+    }
+    return videos
+  }
+  const listedVideos = searchResult(videoState?.videos, videoState.keyword)
+  useEffect(() => {
+    ;(async () => {
+      try {
```

Comment:
I think you forgot to remove semi colon here

Reply:
This is the issue with prettier , I need to check this